python/grpc/demo_grpc.py

```diff
@@ -0,0 +1,19 @@
+from ddtrace import Pin, patch
+import grpc
+
+from hello_pb2 import HelloRequest
+from hello_pb2_grpc import HelloStub
+
+# Use a pin to specify metadata related to this connection
+Pin.override(grpc, service='demo.grpc')
+
+def run():
+    # If not patched yet, you can patch grpc specifically
+    patch(grpc=True)
+    with grpc.insecure_channel('localhost:50051') as channel:
+        stub = HelloStub(channel)
+        response = stub.SayHello(HelloRequest(name="test"))
+        print response
```

Comment:
Are you using Python 2 or 3?

We probably want this as `print(response)` so it will be Python 3 compatible.

If you wanted to you could add `from __future__ import print_statement` at the top to really make sure it is Python 2 and Python 3 compatible, but for fun reasons it isn't required here.

In Python 3, `print()` is a function and needs the parenthesis, in Python 2 it is a statement and therefore does not need them.

However, if you do `print(response)` in Python 2, since it ignores the whitespace when parsing it is the same as writing `print (response)` which is basically "print this grouped statement".

The more you know...